src/Context/SoapContext.php

```diff
@@ -273,4 +273,15 @@ public function beforeStepCheckForException(BeforeStepScope $scope)
             throw $this->fault;
         }
     }
+
+    /**
+     * @Given I register the following headers:
+     */
+    public function iRegisterHeaders(TableNode $headers)
+    {
+        foreach ($headers->getRowsHash() as $key => $value) {
+            $soapHeaders[] = new \SoapHeader($key, $key, $value);
```

Comment:
We can do multiple iterations, no problem, could you also provide docs for this step with a usage example?
Also, can we make rowsHash instead of columns, I think | key | value | is more commonly used and better to understand, also I would make the | namespace | column optional and preset a default namespace if not specified. Do you think this is feasible?